package/kernel/linux/modules/video.mk

```diff
@@ -503,6 +503,23 @@ endef
 
 $(eval $(call KernelPackage,drm-i915))
 
+define KernelPackage/drm-ivpu
+  SUBMENU:=$(VIDEO_MENU)
+  TITLE:=Intel VPU DRM support
+  DEPENDS:=@TARGET_x86 +ivpu-firmware
+  KCONFIG:=CONFIG_DRM_ACCEL_IVPU \
+       CONFIG_DRM_ACCEL_HABANALABS=n \
+       CONFIG_DRM_ACCEL_QAIC=n \
```

Comment:
`CONFIG_DRM_ACCEL_HABANALABS` and `CONFIG_DRM_ACCEL_QAIC` should not be set here at all, you probably want to set all of them to n in the generic config as `CONFIG_DRM_ACCEL` will probably make the buildtime prompt for them

Reply:
There are no default values defined for CONFIG_DRM_ACCEL_HABANALABS and CONFIG_DRM_ACCEL_QAIC, once CONFIG_DRM_ACCEL is on, they should be given explict value. So I am going to set them as "n" in target/linux/x86/64/config-6.6 (they depend on X86_64), is it the right direction?

Reply:
Seems I didn't get the mechanism for kmod package right, need to figure the right way out